app/modules/entities/views/publisher_layout.js

```diff
@@ -20,6 +20,13 @@ export default TypedEntityLayout.extend({
 
   initialize () {
     this.model.initPublisherPublications()
+    this.displayMergeSuggestions = app.user.hasAdminAccess
+  },
+
+  serializeData () {
+    return _.extend(this.model.toJSON(), {
+      displayMergeSuggestions: this.displayMergeSuggestions
+    })
```

Comment:
this method definition is not needed, as the view inherits `serializeData` from `TypedEntityLayout`